.github/workflows/dev-pipeline.yml

```diff
@@ -1,20 +1,58 @@
-name: Process dev-2.x push or pr
+name: Process dev-2.x push
 on:
   push:
     branches:
       - dev-2.x
-  pull_request:
-    branches:
-      - dev-2.x
 jobs:
-  docker-push:
-    if: github.ref == 'refs/heads/dev-2.x'
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: 21
+          distribution: temurin
+          cache: maven
+
+      - name: Run tests
+        run: |
+          mvn test -P prettierSkip
+
+  container-image:
     runs-on: ubuntu-latest
+    needs:
+      - test
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Build docker image from dev-2.x and push it
-        run: ./.github/workflows/scripts/build_and_push_dev.sh
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: 21
+          distribution: temurin
+          cache: maven
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+      - name: Build container image with Jib, push to Dockerhub
         env:
-          DOCKER_USER: ${{ secrets.DOCKER_USER }}
-          DOCKER_AUTH: ${{ secrets.DOCKER_AUTH }}
+          CONTAINER_REPO: docker.io/hsldevcom/opentripplanner:v2
+          CONTAINER_REGISTRY_USER: ${{ secrets.DOCKER_USER }}
+          CONTAINER_REGISTRY_PASSWORD: ${{ secrets.DOCKER_AUTH }}
+        run: |
+          otp_version="v2"
+          commit_hash=`git rev-parse --short "$GITHUB_SHA"`
+          date_time=`date +"%Y-%m-%dT%H-%M-%S"`
+          docker_tag_long="$otp_version-$date_time-$commit_hash"
+
+          mvn --batch-mode -P prettierSkip compile com.google.cloud.tools:jib-maven-plugin:build -Djib.to.tags=$docker_tag_long
```

Comment:
Does this push only date tagged OTP  image? The rest of the  new dataload process seems to require standard otp tags v3 and v3-prod.